alerting/src/aa-storm-alert/worker.ts

```diff
@@ -0,0 +1,79 @@
+import { createConnection } from 'typeorm';
+import { AnticipatoryActionAlerts } from '../entities/anticipatoryActionAlerts.entity';
+import {
+  buildEmailPayloads,
+  filterOutAlreadyProcessedReports,
+  getLatestAvailableReports,
+  transformReportsToLastProcessed,
+} from './alert';
+import { sendStormAlertEmail } from '../utils/email';
+import { ILike } from 'typeorm';
+
+// TODO: for later, we need to support multiple countries
+export const COUNTRY = 'mozambique';
+
+/**
+ * Executes the process to handle anticipatory action alerts.
+ * This script connects to a remote database, retrieves the latest available reports,
+ * filters out already processed reports, prepares email payloads based on alerts,
+ * sends email alerts, and updates the last processed states in the database.
+ *
+ * @returns {Promise<void>} Promise that resolves when all operations are completed.
+ *
+ * @throws {Error} If no alert is found for the specified country in the database.
+ */
+export async function run() {
+  // create a connection to the remote db
+  const connection = await createConnection();
+
+  const alertRepository = connection.getRepository(AnticipatoryActionAlerts);
+
+  const latestAvailableReports = await getLatestAvailableReports();
+
+  // get the last alert which has been processed for email alert system
+  const alert = await alertRepository.findOne({
```

Comment:
@Max-Z80 @Doniaab let's make sure we support multiple alerts here. This would allow us to handle different user groups more easily